logger.py

```diff
@@ -23,5 +23,5 @@ def configure_logger(lvl: int) -> logging.Logger:
     return logger
 
 
-prev_handler: Optional[logging.StreamHandler[TextIO]] = None
+prev_handler: Optional['logging.StreamHandler[TextIO]'] = None
```

Comment:
This is broken in master and throws an error. This fixes the type error but please take a look at this separately

Reply:
@SalDaniele PTAL

Reply:
For this you would need 3.11 python. The quotes might be a way to make it work with older python versions.

Reply:
I am using python3.9. This way works with it. Might want to consider adding it in.

Reply:
verified works for 3.11  and 3.9. We probably want this since it works for both versions